test_fakeredis.py

```diff
@@ -4403,5 +4403,42 @@ def test_hscan_iter(self):
             self.redis.hscan_iter('name')
 
 
+class TestPubSubConnected(unittest.TestCase):
+
+    def setUp(self):
+        self.redis = fakeredis.FakePubSub(connected=False)
```

Comment:
I would rather call this variable `self.pubsub` since it is not an instance of a Redis object.